Rust.gitignore

```diff
@@ -5,3 +5,6 @@
 # Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
 # More information here http://doc.crates.io/guide.html#cargotoml-vs-cargolock
 Cargo.lock
+
+# These are backup files generated by rustfmt
+*.rs.bk
```

Comment:
The [official gitignore](https://github.com/rust-lang-nursery/rustfmt/commit/5ac67adb409f374aa37a40860649f8c770ecdee8) for rustfmt has a slightly different rule: `**/*.rs.bk` - want to mimic that here?